phantomgames/Home/Categories/CategoryViewController.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  CategoryViewController.swift
+//  phantomgames
+//
+//  Created by Aphiwe Shozi on 2024/05/15.
+//
+
+import UIKit
+
+class CategoryViewController: UIViewController {
```

Comment:
empty file again??

Reply:
I put a warning in the repository file, in future should I put a warning on every file associated with the feature